packages/flutter/lib/src/material/snack_bar.dart

```diff
@@ -304,10 +304,11 @@ class SnackBar extends StatefulWidget {
   /// The amount of padding to apply to the snack bar's content and optional
   /// action.
   ///
-  /// If this property is null, then the default depends on the [behavior] and
-  /// the presence of an [action]. The start padding is 24 if [behavior] is
-  /// [SnackBarBehavior.fixed] and 16 if it is [SnackBarBehavior.floating]. If
-  /// there is no [action], the same padding is added to the end.
+  /// If this property is null, the vertical padding's default value will be 14,
```

Comment:
This may be easier to read with if it broke down each padding aspect, like:

If this property is null, the default padding values are
- vertical..
- horizontal..
- etc.

I think it would also be beneficial to mention if it is padding around the content, the action or both, since that was difficult to understand while we were going back and forth on this. :)

Reply:
@Piinks 

I updated the documentation. 7f56a9a

Just realized that the `padding` implementation is somewhat complicated to explain because both `content` and `action` uses the same `padding` property.